Add iaas call 2024-02-14 (#462)
SovereignCloudStack/minutes: iaas/20240214.md

# 2024-02-14

## Participants

- @lindenb1
- @ignatov17
- @o-otte
- @flyersa
- @jschoone
- @fdobrovolny
- @josephineSei
- @matfechner
- @gndrmnn
- @yeoldegrove
- @NotTheEvilOne
- @sbstnnmnn
- @garloff (traveling)
- @matusjenca2
- @martinmo
- @b1-luettje
- @horazont
- @mbuechse
- @brueggemann
- @akafazov

## Housekeeping & Organizational Items

- Report in the community call by: @gndrmnn
- Responsible for the minutes (writing and transfer to github): @jschoone

## Agenda

:rotating_light: When adding items to the agenda, please note:

- Please note your handle
- Please note the amount of time the topic needs in order to be discussed adequately
- If your topic requires certain people from our community to be around, please make sure they're present for the meeting

### Encryption between user workloads (@fdobrovolny)

- 10 minutes
- Do we already have experience in doing this?

> - We want to create a plugin into Neutron using eBPF to secure the traffic automatically between virtual machines :bee: 

Author: `for` sounds better, thanks :)

- SecuStack has experience in encrypting traffic
    - using special SmartNICs which does the encryption, because those are dedicated hardware, not tied to Openstack
    - alternative approach in software (macsec) exists, incomplete
- How does the attacker model looks like?
    - basically the user doesn't have encrypted traffic between workloads
    - someone gaining software access on the compute node the attacker probably has also access to the private key and can decrypt the whole traffic
    - AI @fdobrovolny: create specific attacker models until next meeting
- AI VP04: Check Geneve/VXLAN encryption in OVN
           Encrypting OVN tunnels with IPsec 
           rfc8926 ->  6.1.1. Inter-Data Center Traffic 

### Standard concerning standard images (@mbuechse)

- 10 minutes
- see PR [standards/#472](https://github.com/SovereignCloudStack/standards/pull/472)
    - remove hard-coded least quotas
    - mention fair-use policy instead
- Would this work for you? Could this be stabilized soon?
    - no objections, will proceed to merge this PR, then return to the stabilization proposal

### Future of ceph day-2 tooling (@yeoldegrove)

- 5-10 minutes

> - after the ongoing discussions about weather to choose a the k8s-native approach with rook we did some research and groundwork

Reviewer: ```suggestion
- after the ongoing discussions about whether to choose the k8s-native approach with rook we did some research and groundwork
```

> - after the ongoing discussions about weather to choose a the k8s-native approach with rook we did some research and groundwork

Reviewer 2: ```suggestion
- after the ongoing discussions about wether to choose a the k8s-native approach with rook we did some research and groundwork
```

>   - no official solution for a real migration  to rook

Reviewer: ```suggestion
  - no official solution for a real migration to rook
```
